layouts/default/README.md

```diff
@@ -0,0 +1,28 @@
+<!--
+  ~ Copyright 2018 Adobe. All rights reserved.
+  ~ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License. You may obtain a copy
+  ~ of the License at http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software distributed under
+  ~ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  ~ OF ANY KIND, either express or implied. See the License for the specific language
+  ~ governing permissions and limitations under the License.
+  -->
```

Comment:
Please take out the copyright header. We don't want to claim copyright on this anyway (it's fine in the public domain), it makes the file harder to read, and developers will be confused if they are allowed to edit this file, which is counter-productive.

Reply:
we should use the pure apache copyright header. but it's a good example that people should put one there.